packages/office-ui-fabric-react/src/components/Facepile/Facepile.visualtest.ts

```diff
@@ -0,0 +1,18 @@
+import { Casper, IPhantomCSS } from '../../visualtest/PhantomCssInterface';
+import { baseUrl } from '../../common/VisualTest';
+declare var phantomcss: IPhantomCSS;
+declare var casper: Casper;
+/* tslint:disable:no-function-expression */
+casper.
+  start(baseUrl + 'facepile').
+  then(function () {
+    phantomcss.screenshot('.Facepile', 'Facepile_not_pressed');
+  }).then(function () {
```

Comment:
Facepile doesn't have hover/clicked states. Let's remove these tests. We could use tests that include the action buttons though 
![image](https://cloud.githubusercontent.com/assets/1434956/25141419/5d30e7a6-2418-11e7-9749-97ab3feed96d.png)